nf_core/pipeline-template/.github/workflows/config_tests.yml

```diff
@@ -0,0 +1,62 @@
+name: Test pipeline configurations
+
+# Run the workflow when:
+#  - dispatched manually
+#  - when a PR is opened or reopened to main/master branch
+#  - the head branch of the pull request is updated, i.e. if fixes for a release are pushed last minute to dev.
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+      - master
+
+env:
+  NXF_ANSI_LOG: false
+
+jobs:
+  test_configs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Nextflow
+        uses: nf-core/setup-nextflow@v2
+        with:
+          version: "24.10.1"
+
+      - name: Check out pipeline code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        with:
+          fetch-depth: 0
+
+      - name: Check the correct default config base of the nf-core configs
+        uses: GuillaumeFalourd/assert-command-line-output@v2
+        with:
+          command_line: nextflow config -value params.custom_config_base .
+          contains: https://raw.githubusercontent.com/nf-core/configs/master
+          expected_result: PASSED
+
+      - name: Check the correct inclusion of an existing institutional profile
+        uses: GuillaumeFalourd/assert-command-line-output@v2
+        with:
+          command_line: nextflow config -profile self_hosted_runner -o flat . # We will hopefully always have this profile in configs, even if some HPCs are retired.
```

Comment:
I wouldn't assume we will always have this profile, specially now that we are trying a new provider of the self hosted runners. Maybe `awsbatch` is a safest option?

Reply:
But would that mean that the profile could get renamed? I liked that it was a generic name like the pipelines of nf-core. But fair enough, I switched.

Reply:
Resolved by removal.